src/templates/rails/config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  unless Rails.env.production?
+    get 'styleguide' => 'styleguide#index'
```

Comment:
__[RuboCop]__ Style/StringHashKeys: Prefer symbols instead of strings as hash keys.

<sub>posted by Sider</sub>